.github/workflows/release-wc-and-playground.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
         with:
-          ref: master
+          ref: ${{ github.event.release.target_commitish }}
```

Comment:
just curious what value is it? can you add a comment?

Reply:
Added